### PR TITLE
S2 create account

### PR DESCRIPTION
**Pull request by sudokufan**
Ability to create new accounts and add to mongoDB

### CODE REVIEW

File: Services/AccountsService.js [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
-    var collection = db.collection('accounts')
+    let collection = db.collection('accounts')
     collection.find({}).toArray((err, docs) => {
         callback(docs)
     })
 }
 
-module.exports.getAccounts = getAccounts
\ No newline at end of file
+const createAccount = (db, newAccountInfo, cb) => {
```
Comment (avocadobelly) [inline Services/AccountsService.js:8]: Pick one (cb or callback) for naming parameters, you have used both in idfferent places

**Review by avocadobelly — COMMENTED** [review]
Just one small thing for consistency

File: Services/AccountsService.js (hunk 1 above)
Comment (sudokufan) [reply Services/AccountsService.js:8]: done

**Review by avocadobelly — APPROVED** [review]
Nice, all good to go!

**Pull request merged**